packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1122,6 +1132,26 @@ describe('plugin-meetings', () => {
     });
 
     describe('Public Event Triggers', () => {
+      let sandbox;
+      const {ENDED} = CONSTANTS;
+
+      beforeEach(() => {
+        const fakeMediaTrack = () => ({stop: () => {}, readyState: ENDED});
+
+        sandbox = sinon.sandbox.create();
+        sandbox.stub(Media, 'stopTracks').returns(Promise.resolve());
+        sandbox.stub(meeting.mediaProperties, 'audioTrack').value(fakeMediaTrack());
+        sandbox.stub(meeting.mediaProperties, 'videoTrack').value(fakeMediaTrack());
+        sandbox.stub(meeting.mediaProperties, 'shareTrack').value(fakeMediaTrack());
+        sandbox.stub(meeting.mediaProperties, 'remoteAudioTrack').value(fakeMediaTrack());
+        sandbox.stub(meeting.mediaProperties, 'remoteVideoTrack').value(fakeMediaTrack());
+        sandbox.stub(meeting.mediaProperties, 'remoteShare').value(fakeMediaTrack());
+      });
+      afterEach(() => {
+        sandbox.restore();
+        sandbox = null;
+      });
+
```

Comment:
Also something I ran into, a lot of tests do this:

```javascript
someObject.someProperty = sinon.stub().returns(Promise.resolve());
```

which is okay except it doesn't restore the value and it was throwing off other tests further down. I think using sinon sandbox is great way to stub or mock what you need and then restore values after.

Or at least capture the stub to restore it:

```javascript
const somePropertyStub = sinon.stub(someObject, 'someProperty').returns(Promise.resolve());
// ... later on
somePropertyStub.restore();
```

anyhow just my two cents.